themes/default/content/docs/guides/automation-api/_index.md

```diff
@@ -86,6 +86,10 @@ Visit the [examples repo](https://github.com/pulumi/automation-api-examples) for
 
 Like all of Pulumi, Automation API is available in multiple languages, so you can create applications that use it in TypeScript/JavaScript, Python, Go, and C#.
 
+{{% notes type="info" %}}
+Automation API supports cross-language implementations where it runs in a program of a different language than the Pulumi programs it manages.
```

Comment:
wondering if this requires a note style? can it be worked into the paragraph information instead?

Reply:
I can drop the <Note> style. Will update.